routers/api/v1/org/team.go

```diff
@@ -256,7 +256,7 @@ func EditTeam(ctx *context.APIContext) {
 
 	form := web.GetForm(ctx).(*api.EditTeamOption)
 	team := ctx.Org.Team
-	if err := team.GetUnits(); err != nil {
+	if err := team.LoadUnits(ctx.Context); err != nil {
```

Comment:
I think just `ctx` is OK.

Reply:
Fixed in https://github.com/go-gitea/gitea/pull/22970/commits/c40582704303c30c3481c57bf3c561eb0235afce

Reply:
Ah, interesting.
I saw that as well, but from my look in the code, I didn't know whether it implements `context.Context`, so I accepted it.